### PR TITLE
Adding Defi Oracle Meta mainnet 138

### PR DESCRIPTION
**Pull request by defi-oracle**
## Description

_Concise description of proposed changes_

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)  **Complies with all the must-haves.**

Your ENS/address: defi-oracle.eth

### CODE REVIEW

File: packages/chains/src/defi-oracle-meta.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+  },
+  blockExplorers: {
+    etherscan: {
+      name: 'EhherScan',
+      url: 'https://etherscan.io',
+    },
+    default: {
+      name: 'Quorum Explorer',
+      url: 'https://public-0138.defi-oracle.io',
```
Comment (tmm) [inline packages/chains/src/defi-oracle-meta.ts:33]: This URL doesn't resolve.

**Review by tmm — CHANGES_REQUESTED** [review]
Please add a changeset (`pnpm changeset`) and export the chain from the `index.ts` file.

**Comment by jxom** [comment]
Closing for now as the chain data is malformed (Block Explorer URL does not resolve).

Can you please re-open a separate PR with the requested changes and make sure data is correct and working?

**Comment by defi-oracle** [comment]
I do not see a chain/rpc issue

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request closed without merging**